### DISCUSSION TITLE
Dockerless upgrade

### DISCUSSION THREAD

**Discussion by marcusyoung** [bookwyrm-social/bookwyrm#2396] [General]
Just thinking about the upgrade process for a Dockerless install as that isn't documented. I'd like to understand what manage.py migrate actually does?

**Comment by mouse-reeve** [comment]
`manage.py migrate` is part of Django, not Docker. When changes are made to [the Django models](https://github.com/bookwyrm-social/bookwyrm/tree/main/bookwyrm/models), the migrations update the Postgres database to reflect those changes. You can see the actual migrations in [`bookwyrm/migrations`](https://github.com/bookwyrm-social/bookwyrm/tree/main/bookwyrm/migrations).

**Reply by marcusyoung** [reply]
Thanks. Yeah, I knew it was part of Django, just wasn't sure exactly what it did.
